notebooks/ivf_flat_example.ipynb

```diff
@@ -0,0 +1,803 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.            add_data_on_build=True
</pre><p>For users who might not be coming to this notebook from the blog, it would be helpful to provide a link to the pylibraft docs in a text description above the code and maybe mention that some of these are described in more detail later in the notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/raft/pull/1758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Good point, added.</p>